pom.xml

```diff
@@ -93,9 +99,33 @@
 
     <dependencies>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
It is JUnit version 5. Since KV uses junit5 and we need tests from KV we need junit5, also there is backward compatibility to jnuit4, we can migrate to junit5 pretty easy. 
Also, junit5 has many new useful features